feedback/analyze-feedback.md

```diff
@@ -0,0 +1,223 @@
+
+When identifying issues, don’t rely on volume of feedback alone to prioritize improvements. Feedback submitted may be from people who face cultural, linguistic, geographical, disability, technological, socioeconomic, or other barriers to uptake.
+
+You can seek confirmation of feedback insights using other data sources such as: web analytics, call volumes, social media trends, and GC Task Success Survey results.
+
+While it is best to read the full dataset of feedback, sampling a smaller set of feedback can help identify trends when dealing with more feedback than you can read. If you continue to receive more feedback than you can read, it is best to remove the tool until you have completed your updates.
+
+## Research questions shape analysis
+
+Start by thinking in terms of research questions and who you will be sharing your findings with. Doing this in advance can help clarify how to group feedback when doing manual analysis.
+
+Common research questions:
+
+* What are the most common issues being reported?
+* Are there specific reasons for failure or specific suggestions to improve the experience?
+* What pages are receiving the most feedback?
+* Has feedback increased or decreased after a page update?
+* What types of issues were the most common (findability, comprehension, technical)?
+
+## Manual analysis methods
+
+Grouping feedback with similar issues together with tags is useful for both small and large datasets. It helps you be more efficient with analysis by having smaller sets of data to analyse.
+
+A small dataset may only need a few tags to make sense of the feedback. A large dataset may require two levels of tags to understand specific content issues.
+
+### Best practices for choosing how to group and tag feedback
+
+<details>
+  <summary>Familiarize yourself with your data</summary>
+  <p>Read through a sample of feedback data and try to spot recurring patterns. Jot them down to get a rough overview of WHAT tasks, topics, or issues people are talking about.</p>
+  <p>Not every comment will be useful - sometimes it will be too unclear to use or be completely about another topic.</p>
+</details>
+
+<details>
+  <summary>Consider tags based on a task or issue</summary>
+  <p><strong>Task-based tags</strong> are recommended when analyzing feedback for a group of pages where there are multiple user tasks.</p>
+  <p>To identify tasks, ask yourself why the user came to the site. What were they trying to do, or what question were they trying to answer? </p>
+  <p><strong>Issue-based tags</strong> may be a better strategy when gathering feedback on a single page, single topic, or where a single task dominates your feedback.</p>
+  <p><strong>For large datasets</strong> you may find a second level of tags is needed to add precision. This can be done at the same time you tag the feedback OR when you are ready to analyze a smaller set of feedback.</p>
```

Comment:
```suggestion
  <p><strong>For large datasets</strong> you may find a second level of tags is needed to add precision. This can be done at the same time you tag the feedback <span class="text-uppercase">or</span> when you are ready to analyze a smaller set of feedback.</p>
```